fetch.bs

```diff
@@ -5227,6 +5731,10 @@ constructor must run these steps:
  <a for=request>credentials mode</a> to
  <var>credentials</var>.
 
+ <li><p>If <var>useReferredTokenBinding</var> is true, set <var>request</var>'s
+ <a for=request>referred-token-binding origin</a> to
+ <var>origin</var>.
```

Comment:
This makes it possible for a request's referred-token-binding origin to be set even if no Token Binding key exists for that origin. This seems a bit weird to me.

Reply:
The intent here is to allow scripts to easily indicate that they want to use referred token binding, without letting them set the referred-token-binding origin. This origin is relevant only when TB is being used and there is a TB key.

(Moot now: API changes have been removed.)